src/js/models/PluginOptions.ts

```diff
@@ -12,6 +12,7 @@ export const pluginOptions: IPluginOptions = {
   },
   translation: {
     autoTranslate: true,
+    translateWholePage: true,
```

Comment:
Tā kā WEB-T projektā arī izmanto šo spraudni, vajadzētu ieslēgt šo tikai, ja API versija > 2, citādi izmaiņas skars arī WEB-T

Reply:
WebsiteTranslator.Options.api.version

Reply:
ieskaitot versiju 2, vai tikai sākot no v3?

Reply:
Sākot ar v3

Reply:
pievienoju index.ts, ka ja versija ir zem 2 (ieskaitot 2) un whole page translation ir ieslēgts, tad tas tiek izslēgts. visādi citādi, ieslēgšana un izslēgšana darbojas normāli

Reply:
Bet tagad sanāks, ka v2 nav iespējams ieslēgt manuāli to funkciju, tas arī nav pareizi

Reply:
aizstāju, ka tikai v1 tad tā funkcija tiek vienmēr atslēgta

Reply:
Nevajag nevienā versijā, lai ir vienmēr atslēgta. Vajag, lai v1 un v2 pēc noklusējuma ir izslēgta (bet var ieslēgt); no v3 uz augšu - pēc noklusējuma ieslēgta (bet var izslēgt).

Reply:
tad varbūt atsevišķus flags uztaisīt?